### PR TITLE
Geographic traffic routing method in Azure Traffic Manager

### PR DESCRIPTION
**Pull request by vladca**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [X ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [X ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [X ] Title of the pull request is clear and informative.
- [X ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as a link to the swagger spec, used to generate the code.
- [ ] The `project.json` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by msftgits** [comment]
Closing and reopening to ping CLA bot.

**Comment by markcowl** [comment]
@vladca traffic manager tests are failing

**Comment by markcowl** [comment]
@vladca Why are you making changes to the master branch, instead of the AutoRest branch?

File: src/ResourceManagement/TrafficManager/TrafficManagerManagement/Generated/EndpointOperations.cs [hunk 1]
```diff
@@ -154,7 +154,7 @@ public TrafficManagerManagementClient Client
             url = url + "/";
```
Comment (markcowl) [inline src/ResourceManagement/TrafficManager/TrafficManagerManagement/Generated/EndpointOperations.cs]: Where are the tests for the new features?

**Review by markcowl — CHANGES_REQUESTED** [review]
Need tests for the new features, and need to re-record existing tests for the new api version
[Vlad]: Done!

File: src/ResourceManagement/TrafficManager/TrafficManagerManagement/Generated/EndpointOperations.cs (hunk 1 above)
Comment (vladca) [inline src/ResourceManagement/TrafficManager/TrafficManagerManagement/Generated/EndpointOperations.cs]: Added

**Comment by vladca** [comment]
This is done now, thank you Mark

File: src/ResourceManagement/TrafficManager/TrafficManagerManagement/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -20,8 +20,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Traffic Manager Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure Traffic Manager management functions for managing the Microsoft Azure Traffic Manager service.")]
 
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
+[assembly: AssemblyVersion("1.1.0.0")]
```
Comment (markcowl) [inline src/ResourceManagement/TrafficManager/TrafficManagerManagement/Properties/AssemblyInfo.cs]: AssemblyVersion shopuld remain at 1.0.0.0 to maintain backward compatibility, AssemblyFileVersion should match the package version (as it does).

**Review by markcowl — CHANGES_REQUESTED** [review]

File: src/ResourceManagement/TrafficManager/TrafficManagerManagement/Properties/AssemblyInfo.cs (hunk 2 above)
Comment (ms-alguerra) [inline src/ResourceManagement/TrafficManager/TrafficManagerManagement/Properties/AssemblyInfo.cs]: This is now fixed

**Comment by cormacpayne** [comment]
@vladca it looks like you changed the `AssemblyFileVersion` when you only needed to change `AssemblyVersion`. Please change the `AssemblyFileVersion` back to 1.1.0.

**Review by markcowl — APPROVED** [review]

**Pull request merged**